Manifest.toml

```diff
@@ -0,0 +1,2 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Do we need to include the .toml file for release? if not let's get rid of it

Reply:
Good spot, gonna need to remove that file!